### DISCUSSION TITLE
How to list operators used by particular ONNX model?

### DISCUSSION THREAD

**Discussion by OrdevAg** [microsoft/onnxruntime#11952] [Other Q&A]
I'm trying to get acceleration of Random Forest model converted from `sklearn` to `onnx` using NNAPI execution provider. There is a list of supported operators: https://onnxruntime.ai/docs/execution-providers/NNAPI-ExecutionProvider.html#supported-ops
I'd like to see what operators that are used in my model are not supported by NNAPI execution provider. How do I get such list? Or is there a way to list all operators used by my model?

**Comment by RyanUnderhill** [comment]
OnnxRuntime should tell you what operators couldn't be implemented in a provider if you set your logging level to ORT_LOGGING_LEVEL_VERBOSE (or maybe ORT_LOGGING_LEVEL_INFO I'm not 100% sure) when creating the OrtEnv. If all work in the one provider it'll just say that all are using the same provider.

But this should tell you what you need.
